packages/commonwealth/server/socket/createNamespace.ts

```diff
@@ -0,0 +1,102 @@
+import { Server } from 'socket.io';
```

Comment:
Or... were they just renamed? If so, can we not lint them, so it's at least clear they were renamed and not rewritten?